Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/people-dashboards.md

---
id: people-dashboards
title: People Dashboards
sidebar_label: People
description:

>   People is a parachain on Polkadot focused on decentralized identity and social interactions,
>   enabling users to manage their digital identity and engage in community governance.

Reviewer: ```suggestion
  People's Chain focuses on decentralized identity and social interactions,
  enabling users to manage their digital identity and engage in community governance.
```

keywords: [polkadot, dashboard, dune, people, identity]
slug: ../people-dashboards
---

# People Dashboards

## Overview

> People is a parachain on Polkadot focused on decentralized identity and social interactions,
> enabling users to manage their digital identity and engage in community governance.

Reviewer: ```suggestion
People's Chain focuses on decentralized identity and social interactions,
enabling users to manage their digital identity and engage in community governance.
```

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the People parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the People parachain:
```
